README.md

```diff
@@ -72,6 +72,8 @@ type Config = {
   url: string;
   /** Pattern to match against for links on a page to subsequently crawl */
   match: string;
+  /** Optional Selector to limit grabbing the links from */
+  matchSelector?: string;
```

Comment:
the term `matchSelector` is a bit broad, maybe we can call this `linkSelector` to be more clear whats different between this and `selector`

Reply:
thanks @steve8708 for feedback
my reason for selecting "matchSelector" was to make it tied to the "match" property.
on the other hand, you have a point too.
what do you think?